docs/users/Shared_Configurations.mdx

```diff
@@ -188,6 +188,11 @@ See [`configs/strict.ts`](https://github.com/typescript-eslint/typescript-eslint
 We recommend a TypeScript project extend from `plugin:@typescript-eslint/strict` only if a nontrivial percentage of its developers are highly proficient in TypeScript.
 :::
 
+:::note
```

Comment:
```suggestion
:::warning
```
Same for below

Reply:
I'd rather not make this stand out quite so much. The warning color is ... a lot. And we already mention this at the top of the file. @Josh-Cena how strongly do you want this? 😄

Reply:
I think this basically means "if you use it it will break", so yes it's quite serious. I think yellow is "just right" for this.